docs/source/deploying/prefixed-root.md

```diff
@@ -0,0 +1,68 @@
+---
+html_meta:
```

Comment:
```suggestion
myst:
  html_meta:
```

Also indent lines 3-6 with two more spaces. The format of this changed since this PR was created.